src/components/pricing/index.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react'
+import {FunctionComponent} from 'react'
+import SelectPlanNew from 'components/pricing/select-plan-new'
+import Testimonials from 'components/pricing/testimonials'
+import testimonialsData from './testimonials/data'
+import PoweredByStripe from './powered-by-stripe'
+
+const QUANTITY_PRICING_ENABLED =
+  process.env.NEXT_PUBLIC_FF_QUANTITY_PRICING_AVAILABLE !== 'false'
+
+type PricingProps = {
+  viewer: any
+  authToken: string
+  prices:
+    | any
+    | {
+        annualPrice: {
+          id: string
+          recurring: {
+            interval: 'year'
+          }
+          unit_amount: number
+        }
+      }
+  pricesLoading: boolean
+  needsEmail: boolean
+  setNeedsEmail: React.Dispatch<React.SetStateAction<boolean>>
+  redirectURL?: string
+  dark?: boolean
```

Comment:
suggestion: passing `dark` in should probably be removed in favor of a global storybook level toggle imo

Reply:
done that in this PR: https://github.com/eggheadio/egghead-next/pull/381